rules/windows/c2_grey_list_beacon_sequence.toml

```diff
@@ -0,0 +1,88 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network activity from unexpected system applications. This may indicate adversarial activity as these
+applications are often leveraged by adversaries to execute code and evade detection.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Unusual Process Network Connection: Sequence"
+risk_score = 21
+rule_id = "1fe3b299-fbb5-4657-a937-1d746f2c711a"
+severity = "medium"
+tags = ["from-eql-rules"]
+type = "eql"
+
+query = '''
+sequence by process.entity_id, process.pid, process.name with maxspan=5m
+  [process where event.type in ("start", "process_started") and
+
+     // known applocker bypasses
+     process.name in ("bginfo.exe",
+                      "cdb.exe",
+                      "control.exe",
+                      "cmstp.exe",
+                      "csi.exe",
+                      "dnx.exe",
+                      "fsi.exe",
+                      "ieexec.exe",
+                      "iexpress.exe",
+                      "installutil.exe",
+                      "Microsoft.Workflow.Compiler.exe",
+                      "msbuild.exe",
+                      "msdt.exe",
+                      "mshta.exe",
+                      "msiexec.exe",
+                      "msxsl.exe",
+                      "odbcconf.exe",
+                      "rcsi.exe",
+                      "regsvr32.exe",
+                      "xwizard.exe")]
+  [network where event.type == "connection"]
+'''
+
+
+[[rule.threat]]
```

Comment:
All of these rules will need the `threat` mappings reviewed/revised. Since both technique and tactic were arrays, all techniques were rolled into each tactic (which may not be the desired relationship)

Reply:
I would pick just a single tactic for these until #51 and its blockers are resolved

Reply:
wfm 👍